### PR TITLE
Projects documentation updates

### PR DESCRIPTION
**Pull request by mphalke**
doc,projects:ad4130_iio: Added github documentation
-Added web based documentation for ad4130 project

doc: Added sphinx folder for documentation resources
-Added initial folder for storing doc resources

### CODE REVIEW

File: doc/sphinx/source/evb_connection_diagram.png
Comment (jansunil) [inline doc/sphinx/source/evb_connection_diagram.png:1]: Should we now add matlab also in the list of IIO clients we've mentioned under the block "IIO Oscilloscope" ?

Comment (mphalke) [reply doc/sphinx/source/evb_connection_diagram.png:1]: yes, make sense. Will update this pic

**Comment by mphalke** [comment]
btw, document is reflected at below link, so if anyone has any comments on actual contents, feel free to add.
https://github.com/analogdevicesinc/precision-converters-firmware/tree/projects-documentation-update/projects/ad4130_iio

File: projects/ad4130_iio/README.rst [hunk 1]
```diff
@@ -0,0 +1,374 @@
+AD4130 IIO Application
+====================
+
+Introduction
+-----------------
+
+This guide gives an overview of using the ARM platform(s) supported (default is Mbed) 
+firmware with Analog Devices AD4130 Evaluation board and SDP-K1 (or other compatible)
```
Comment (ribdp) [inline projects/ad4130_iio/README.rst:8]: should the closing parenthesis be after 'board' in the next line?

File: projects/ad4130_iio/README.rst [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+
+Software Downloads
+-----------------
+
+Below section provides an overview about the pre-requisites needed to build and run the project:
+
+* Source code is hosted on `precision-converters-firmware <https://studio.keil.arm.com/auth/login/>`_ repo
+* `Build Guide for projects <https://wiki.analog.com/resources/tools-software/product-support-software/pcg-fw-mbed-build-guide>`_
+* `Libiio: IIO Library <https://github.com/analogdevicesinc/libiio/releases>`_ 
+This library provides an abstracted library interface to communicate IIO device
```
Comment (ribdp) [inline projects/ad4130_iio/README.rst:108]: nit: should we remove the second instance of the word 'library'?

File: projects/ad4130_iio/README.rst (hunk 1 above)
Comment (mphalke) [reply projects/ad4130_iio/README.rst:8]: I think they are correct. What I meant was:
... AD4130 Evaluation board and SDP-K1 (or other compatible) MCU controller board

 (or other compatible) - This indicates other boards such as STM32-Disco, etc

**Comment by mphalke** [comment]
V1.0: Review comments fixes
1. Corrected evb_connection diagram
2. FIxed typos

**Review by jansunil — APPROVED** [review]

**Review by ribdp — APPROVED** [review]

**Pull request merged**